x-pack/plugins/security_solution/server/lib/detection_engine/rule_management/logic/exceptions/is_valid_exceptions_list.ts

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { ListArray } from '@kbn/securitysolution-io-ts-list-types';
+import { ExceptionListTypeEnum } from '@kbn/securitysolution-io-ts-list-types';
+import type { RulesClient } from '@kbn/alerting-plugin/server';
+import { findRules } from '../search/find_rules';
+
+/**
+ * Util to check if linked exceptions exist already the system
+ * @param exceptionLists {array} - exception lists and items to import
+ * @returns {Promise} exception lists or error if more than one default list found
+ */
+export const isValidExceptionList = async ({
```

Comment:
Let's rename `isValidExceptionList` to something like `validateExceptionList`. We usually start boolean values with the `is` prefix, but not functions.